ibm/service/vpc/resource_ibm_is_vpc.go

```diff
@@ -644,6 +734,47 @@ func vpcGet(d *schema.ResourceData, meta interface{}, id string) error {
 	if err != nil {
 		return err
 	}
+	if vpc.Dns != nil {
+		dnsList := make([]map[string]interface{}, 0)
+		currentDns := map[string]interface{}{}
+		if vpc.Dns.EnableHub != nil {
+			currentDns[isVPCDnsEnableHub] = vpc.Dns.EnableHub
+		}
+		if vpc.Dns.ResolutionBindingCount != nil {
+			currentDns[isVPCDnsResolutionBindingCount] = vpc.Dns.ResolutionBindingCount
+		}
+		if vpc.Dns.Resolver != nil {
+			switch reflect.TypeOf(vpc.Dns.Resolver).String() {
+			case "*vpcv1.VpcdnsResolverTypeDelegated":
+				{
+					resolver := vpc.Dns.Resolver.(*vpcv1.VpcdnsResolverTypeDelegated)
+					currentDnsResolver := map[string]interface{}{}
+					currentDnsResolver[isVPCDnsResolverType] = resolver.Type
+
+					currentDnsResolver[isVPCDnsResolverVpc] = setDnsResolverVpc(resolver.VPC)
```

Comment:
isVPCDnsResolverVpc should be defined in the schema

Reply:
addressed